doc/Options/z390_filetypes.md

```diff
@@ -0,0 +1,32 @@
+CPY | MA | ASCII | Copybook source included in source programs, macros, or other copybooks|Same format as BAL
+DAT | M | ASCII | Text data input files read via AREAD statements in macro processor specifying ID=, DDNAME= or DSNAME= extended parameters.  AREAD with no extended parameters reads the next statement in the input stream.|Same format as BAL except text records can be any length with no continuation format required using ALLOW
+ERR | MALE | ASCII | Output log of all START and ENDED messages plus any errors plus any error file references.|Text file with messages showing start and stop statistics for each step and any error messages issued by those steps.  A cross reference of all macro and copybook files with errors is included at the end.  For full cross reference of all macros and copybooks used see STA file.
+LOG | E | ASCII | Output log of for GUI or batch command execution controlled by CON option|Test file with messages issued udring program execution including start and stop statistics plus any errors and any WTO write to operator messages.
+LKD | L | ASCII | Optional input command for linker (INCLUDE, ENTRY, ALIAS, NAME)|Same format as BAL.
+MAC | M | ASCII | macro source files|Same format as BAL.
+MLC|| M | ASCII | macro assembler source programs read by mz390|Same format as BAL.
+MOD | LE | Binary | Binary code output from linker with no header and no RLD records.  This type file can be created using option MOD and can be loaded as a data table by Micro Focus CICS,|Binary file with all CSECT binary data.  For example DC C'ABC' using option MOD will result in 3 byte MOD file containing x'C1C2C3'.
+OBJ|AL|EBCDIC, ASCII|Output relocatible object code file for macro assembler with two format options. |The default is mainframe linker compatible relocatible object code 80 records (ESD, TXT, RLD, and END).  When OBJHEX option is specified for assembler, ASCII readable text extended format supporting 31 bit long sections is generated.  The z390 linker can read either format by examining file.
```

Comment:
1. Change 2 occurrences of "relocatible" to "relocatable".
2. "... code 80 records ..." -> "... code 80  byte records ...".
3. "... by examining file." -> "... by examining the file.".

Reply:
Resolved by 52df1cf